README.md

```diff
@@ -6,7 +6,148 @@
+```
+
+#### backend specific parameters
+##### TSDB
+* rate 
+* aggregates (optional)
+* aggregation-granularity (optional)
+
+For detailed info on these parameters please visit [TSDB](https://github.com/v3io/v3io-tsdb) docs.  
+Example:
+```python
+client.create('tsdb', '/mytable', attrs={'rate': '1/m'})
+```
+
+##### Stream
+* shards (optional)
+* retention_hours (optional)
+
+For detailed info on these parameters please visit Stream docs.  
+Example:
+```python
+client.create('stream', '/mystream', attrs={'shards': '6'})
+```
+
+### Write
+Writes a Dataframe into one of the supported backends.  
+Common write parameters:
+* dfs - list of Dataframes to write
+* index_cols=None (optional) - specify specific index columns, by default Dataframe's index columns will be used.
+* expression=' ' (optional)
+* condition=' ' (optional)
+* labels=None (optional)
+* max_in_message=0 (optional)
+* partition_keys=None (optional)
+
+Example:
+```python
+data = [['tom', 10], ['nick', 15], ['juli', 14]] 
+df = pd.DataFrame(data, columns = ['name', 'age'])
+df.set_index('name')
+client.write(backend='kv', table='mytable', dfs=df)
+```
+
+### Read
+Reads data from a backend.  
+Common read parameters:
+* query: string - Query in SQL format
+* iterator: bool - Return iterator of DataFrames or (if False) just one DataFrame
+* filter: string - Query filter (can't be used with query)
+* group_by: string - Query group by (can't be used with query)
+* columns: []str - List of columns to pass (can't be used with query)
+* limit: int - Maximal number of rows to return
+* data_format: string - Data format
+* row_layout: bool - Weather to use row layout (vs the default column layout)
+* max_in_message: int - Maximal number of rows per message
+* marker: string - Query marker (can't be used with query)
+
+#### backend specific parameters
+##### TSDB
+* start: string
+* end: string
+* step: string
+* aggragators: string
+* aggregationWindow: string
+
+For detailed info on these parameters please visit [TSDB](https://github.com/v3io/v3io-tsdb) docs.  
+Example:
+```python
+df = client.read(backend='tsdb', query="select avg(cpu) as cpu, avg(diskio), avg(network)from mytable", start='now-1d', end='now', step='2h')
+```
+
+##### KV
+* sharding_keys: []string - list of specific sharding keys to query. For range scan formatted tables only.
+* segments: []int64 (Not yet supported)
+* total_segments: int64 (Not yet supported)
+* sort_key_range_start: string (Not yet supported)
+* sort_key_range_end: string (Not yet supported)
+
+For detailed info on these parameters please visit KV docs.
+
+##### Stream
+* seek: string
+* shard_id: string
+* sequence: int64 
+
+For detailed info on these parameters please visit Stream docs.
+
+### Delete
+Deletes a table of a specific backend.
+
+#### backend specific parameters
+##### TSDB
+* start: string - delete since start
+* end: string - delete since start
+
```

Comment:
can we delete all (like 'tsdb delete -a')?

Reply:
Yes, added to docs